tests/conftest.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+
+@pytest.fixture
+def my_tmp_path(tmp_path):
+    return tmp_path
```

Comment:
This fixture is not doing anything...

Reply:
It is. I need it because I want the tmp_path given to the other fixtures and to the test itself to be the same. This solution seems to me the cleanest.

Reply:
I see your point, but, in order to improve clarity, I'd do the following three things:
1. the easiest: comment why you are defining it in a docstring
2. the most complicate: choose a telling name (e.g. `shared_tmp_path`, but it is still not optimal, according to me)

However, it seems weird to me that you need it, since it should be the same out-of-the-box:
- *This means we can request fixtures in multiple fixtures that are dependent on them (and even again in the test itself) without those fixtures being executed more than once.* https://docs.pytest.org/en/7.2.x/how-to/fixtures.html#fixtures-can-be-requested-more-than-once-per-test-return-values-are-cached
- your fixture is really doing nothing, since if `tmp_path` invoked multiple times were yielding different values, I see no reason why `my_tmp_path` should yield a single one: if multiple values per function are allowed, then multiple `my_tmp_path` invocations would result in multiple `tmp_path` invocations

In general, you can achieve what you want by using a [scope](https://docs.pytest.org/en/7.2.x/how-to/fixtures.html#scope-sharing-fixtures-across-classes-modules-packages-or-session), and to declare a `"module"` or `"class"` scope you would require what you put there (plus the `scope` kwarg, of course). But `"function"` is already the default scope.

Reply:
In general, if you need multiple values per-function, you should pick a factory instead.

Reply:
actually, in this case you could also just use different subdirectories inside `tmp_path`

Reply:
@felixhekhorn why? `tmp_path` is already different for each test